<launch>
    <node name="processor" pkg="project" type="imageProcessor" output="screen" />
    <node name="game_controller" pkg="project" type="controller" output="screen" />
</launch>